File: test/cpp/test_934.cpp
#include "../../src/cpp/code_934.cpp"
#include "cpp_deps/boilerplate.h"

void test(Solution& sol, const json& input, const json& output) {
    vector<vector<int>> grid = input["grid"].get<vector<vector<int>>>();
    int expected = output.get<int>();
    int result = sol.shortestBridge(grid);
    CHECK_EQ(result, expected);
}

TEST_CASE("") {
    TEST("test/test_json/test_934.json");
}

File: test/python/test_934.py
import sys
import os

sys.path.append(os.getcwd())
from .python_deps.boilerplate import Test
from src.python.code_934 import Solution


def input_fetcher(case):
    return case["input"]["grid"]


def test():
    test = Test(934, input_fetcher)
    solution = Solution().shortestBridge
    test.run_tests(solution)
